static/js/common.js

```diff
@@ -58,8 +58,25 @@ function password_quality(password, bar, password_field) {
     return acceptable;
 }
 
+function password_warning(password, password_field) {
+    if (typeof zxcvbn === 'undefined') {
+        return undefined;
+    }
+
+    var min_password_length = 6;
```

Comment:
```
'PASSWORD_MIN_LENGTH': 6,
'PASSWORD_MIN_ZXCVBN_QUALITY': 0.5,
```
These are always defined in the backend settings and filled to `password_field` ? Because I guess it is somewhat like we already have specified a default there and specifying it here again.